### ISSUE TITLE
High Priority - WCAG Level A - Links in the navigation do not receive keyboard focus

### ISSUE THREAD

**Issue by JuliaSang** [w3c/w3c-website#162]
There are links within the navigation that do not receive keyboard focus, so cannot be accessed by users who are navigating using the keyboard and are also not available to screen reader users.

WCAG Reference:
2.1.1 Keyboard (Level A)
[Understanding Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) | [How to Meet Keyboard](https://www.w3.org/WAI/WCAG21/quickref/#keyboard)

2.1.3 Keyboard (No Exception) (Level AAA)
[Understanding Keyboard (No Exception)](https://www.w3.org/WAI/WCAG21/Understanding/keyboard-no-exception.html) | [How to Meet Keyboard (No Exception)](https://www.w3.org/WAI/WCAG21/quickref/#keyboard-no-exception)

1.3.1 Info and Relationships (Level A)
[Understanding Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) | [How to Meet Info and Relationships](https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships)

**Issue ID: DAC_Links_01**

URL: https://beta.w3.org/
Page title: We believe in one web for all – W3C 

Journey 1
<img width="810" alt="links" src="https://user-images.githubusercontent.com/20167528/159245245-f13438b7-a7bf-498e-b383-59a933199d75.png">

The 'Explore web standards' link cannot be tabbed to using the keyboard, nor is it detectable to screen reader users; it can only be accessed by mouse users.

The link has a negative tabindex attribute which takes it out of the natural tab order of the page; providing tabindex="-1" removes the element from the default navigation flow. Elements that must be keyboard navigable, such as a link or button that sighted users can click on with the mouse should not be assigned tabindex="-1".

Additionally the link has been concealed entirely from screen reader users using the aria- hidden attribute.

The 'Standards' link which immediately precedes it visually looks like a heading but is not programmatically determinable as one. Additionally, both links have the same destination.

This is also the case for the following links in each of the submenus in the main navigation and will need to be resolved for each of them:
Groups: Discover W3C groups
Get involved: Find ways to get involved
Resources: Learn from W3C resources 
News and events: Follow news and events 
About: Find out more about us

**Current code ref(s):** #global-nav > div > ul > li:nth-child(1) > div > div > div > div > a

```
<div class="l-center">
  <div class="nav__submenu__intro">
[...]
<a href="/standards/" class="nav-link">Standards</a>
<div class="nav__submenu__intro__text">
<p>Understand the various specifications, their maturity levels on the web
standards track, and their adoption.</p>
<a href="/standards/" aria-hidden="true" tabindex="-1">Explore web standards</a>
    </div>
  </div>
[...] </div>
```

**Keyboard only user comments:**

> “I found when I clicked the expandable link labelled ‘Standards’, it expands and all the links within are accessible, except for the one labelled ‘Explore web Standards’ as it receives no focus indication.
However, the link ‘Standards’ does get focus indication and is clickable. I then got help from someone able to use a mouse to check the link labelled ‘Explore web Standards’, and I found it was clickable and directs me to the same page as the 'Standards' link does.”

**Solution:**
As both links navigate users to the same destination with only one of them being accessible to all users, remove the link mark up from the 'Standards' link, and instead implement this as a heading which introduces the section.

This will leave the 'Explore Web Standards' link as the sole link which navigates users to the web standards page, and will address the issue of duplicated links. In order for this link to be exposed to users that navigate using the keyboard alone and screen reader users respectively, the negative tabindex and aria-hidden attributes will need to be removed.

**Example:**
```
<div class="l-center">
  <div class="nav__submenu__intro">
[...]
    <h2>Standards<h2>
<div class="nav__submenu__intro__text">
<p>Understand the various specifications, their maturity levels on the web
standards track, and their adoption.</p>
<a href="/standards/">Explore web standards</a>
    </div>
  </div>
[...] </div>
```

**Comment by maries24** [comment]
Solution:
!!relates to issue [#155](https://github.com/w3c/w3c-website/issues/155) which, although low priority, needs to be addressed as well if this one is.

As both links navigate users to the same destination with only one of them being accessible to all users, remove the link mark up from the 'Standards' link, and instead implement this as a heading which introduces the section.

This will leave the 'Explore Web Standards' link as the sole link which navigates users to the web standards page, and will address the issue of duplicated links. In order for this link to be exposed to users that navigate using the keyboard alone and screen reader users respectively, the negative tabindex and aria-hidden attributes will need to be removed.

Example:
```
<div class="l-center">
  <div class="nav__submenu__intro">
[...]
    <h2>Standards<h2>
<div class="nav__submenu__intro__text">
<p>Understand the various specifications, their maturity levels on the web
standards track, and their adoption.</p>
<a href="/standards/">Explore web standards</a>
    </div>
  </div>
[...] </div>
```
